__tests__/app.test.js

```diff
@@ -78,18 +78,38 @@ describe("/api/articles/:article_id", () => {
           expect(response.body.msg).toBe("Article id does not exist");
         });
     });
-  })
-})
+  });
+});
 describe("/api", () => {
-  test("GET: 200: responds with object containing all the ap endpoints", () => {
+  test("GET: 200: responds with object containing all the api endpoints", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then((response) => {
-        const endpoints  = response.body;
+        const endpoints = response.body;
         expect(endpoints).toEqual({
-         endpoint
+          endpoint,
         });
       });
-    })
-  })
+  });
+});
+
+describe('/api/articles', () => {
+  test('GET 200   | Returns 200 and an object of all articles with comment_count', () => {
+      return request(app).get('/api/articles').expect(200).then(({body}) => {
+          expect(body.articles.length).not.toBe(0)
+          body.articles.forEach(article => {
+              expect(article).toHaveProperty("author")
+              expect(article).toHaveProperty("title")
+              expect(article).toHaveProperty("body")
+              expect(article).toHaveProperty("topic")
+              expect(article).toHaveProperty("created_at")
+              expect(article).toHaveProperty("votes")
+              expect(article).toHaveProperty("article_img_url")
+              expect(article).toHaveProperty("comment_count")
+          })
+      })
```

Comment:
Great testing so far! Could we also just please test that the response is coming back sorted by date in descending order? You can either add that to this test, or create a new one 🙂